meson.build

```
@@ -21,7 +21,7 @@ project(
     'cpp',
     version: '0.8.0-SNAPSHOT',
     license: 'Apache 2.0',
```

SPDX license expression is preferred.

```suggestion
    license: 'Apache-2.0',
```